trymito.io/pages/use-cases/python-training.tsx

```diff
@@ -0,0 +1,320 @@
+import Image from 'next/image';
+import Footer from '../../components/Footer/Footer';
+import Header from '../../components/Header/Header';
+import pageStyles from '../../styles/Page.module.css';
+import securityStyles from '../../styles/Security.module.css';
+import titleStyles from '../../styles/Title.module.css';
+import pythonTrainingStyles from '../../styles/PythonTraining.module.css';
+import spreadsheetAutomationStyles from '../../styles/SpreadsheetAutomation.module.css';
+import textImageSplitStyles from '../../styles/TextImageSplit.module.css';
+
+// Import Icons & Background Grid
+import { classNames } from '../../utils/classNames';
+import CTAButtons from '../../components/CTAButtons/CTAButtons';
+import customerCardStyles from '../../components/CustomerCard/CustomerCard.module.css';
+import ContactCTACard from '../../components/CTACards/ContactCTACard';
+
+import automationIcon from '../../public/python-training/automation.svg';
+import bugIconPurple from '../../public/python-training/bug_purple.svg';
+import dataIconPurple from '../../public/python-training/data_purple.svg';
+import fastClockIcon from '../../public/python-training/fast_clock.svg';
+import infoIcon from '../../public/python-training/info.svg';
+import React, { useEffect, useState } from 'react';
+
+const PythonTraining: NextPage = () => {
+
+    const [showQuizAnswer, setShowQuizAnswer] = useState(false);
+    useEffect(() => {
+        if (showQuizAnswer) {
+            setTimeout(() => {setShowQuizAnswer(false)}, 3000)
+        }
+    }, [showQuizAnswer])
+    const quizAnswerClass = showQuizAnswer ? pythonTrainingStyles.display_answerquiz_answer : undefined
+
+
+    /* Rotate the final h2 heading to display all of the people that use Mito Python Training */
+    const [currentHeading, setCurrentHeading] = useState(1);
+    const headingIntervalTime = 3000;
+    const headingCount = 10;
+    useEffect(() => {
+        const interval = setInterval(() => {
+            // Save the current heading so we know which one to display next
+            const nextHeadingToDisplay = currentHeading + 1 <= headingCount ? currentHeading + 1 : 1;
+            setCurrentHeading(nextHeadingToDisplay)
+            
+            // Add the display class to the correct heading by id
+            const headingElements = document.getElementsByClassName(pythonTrainingStyles.rotating_heading);
+            console.log(headingElements)
+            for (let i = 0; i < headingElements.length; i++) {
+                const element = headingElements[i];
+                if (element.id === `rotating_header_${currentHeading}`) {
+                    element.classList.add(pythonTrainingStyles.display_rotating_heading);
+                } else {
+                    element.classList.remove(pythonTrainingStyles.display_rotating_heading);
+                }
+            }
+        }, headingIntervalTime);
+        return () => clearInterval(interval);
+    }, [currentHeading])
```

Comment:
This feels like big-time overkill. Do we need to display all the different headers, or do we just want them to be part of the HTML?

If there's not a strong need to actually display them (which, I don't think rotating names like this is going to convince users of anything) -- then let's just get rid of this.

Reply:
Getting rid of this for now and will follow up with team if its needed.